pkg/ebpf/tracee.go

```diff
@@ -820,6 +834,10 @@ func (t *Tracee) getOptionsConfig() uint32 {
 		cOptVal = cOptVal | optForkProcTree // tell sched_process_fork to be prolix
 	}
 
+	if t.config.Output.ExportAnalyze {
```

Comment:
Sounds like exportanalyze should just enable the process tree, instead of having its own logic.

Reply:
You are right :)